templates/facets/facets-item-list.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Override facets item lists, alter title, inject classes.
+ */
+#}
+<div class="facets-widget- {{- facet.widget.type -}} ">
+  {% if facet.widget.type -%}
+    {%- set attributes = attributes.addClass('item-list__' ~ facet.widget.type) %}
+  {%- endif %}
+  {% if facet.widget.type == 'dropdown' -%}
+    {%- set attributes = attributes.addClass('form-select') %}
+  {%- endif %}
```

Comment:
Not sure if we should do this here or in a preprocess. Probably better in a preprocess. In any case, let's add some blocks around to ease overrides from users.

Reply:
Since we already have a check inherited from the parent regarding dropdown, it's ok to keep it here. But let's add blocks.